src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_private_link_scope.py

```diff
@@ -51,6 +52,7 @@ def test_monitor_private_link_scope_scenario(self, resource_group, resource_grou
             'workspace_id': workspace_id
         })
 
+        # this command failed as service cannot find component of application insights
         self.cmd('monitor private-link-scope scoped-resource create -g {rg} -n {assigned_app} --linked-resource {app_id} --scope-name {scope}', checks=[
```

Comment:
Service is always complaining that cannot find a just-created app-insight component. So have to _record_only_ it

```
        if response.status_code not in [200, 201, 202]:
            exp = CloudError(response)
            exp.request_id = response.headers.get('x-ms-request-id')
>           raise exp
E           msrestazure.azure_exceptions.CloudError: Azure Error: PreconditionFailed
E           Message: Resource '/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/clitest.rgfpfgcxn6lts354fiisn66us22nbdcbjihethfyig26yfqbqtg5fmwxxb2jnv224qc/providers/microsoft.insights/components/clitestwqt2popxlabbl' not found
E           Inner error: {
E               "trace": [
E                   "Microsoft.AppInsights.Nexus.ResourceStore.ResourceStoreException"
E               ]
E           }
```